Gemfile

```diff
@@ -8,8 +8,8 @@ gem "rails", "~> 7.1.3", ">= 7.1.3.4"
 # The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem "sprockets-rails"
 
-# Use sqlite3 as the database for Active Record
-gem "sqlite3", "~> 1.4"
+# Use postgresql as the database for Active Record
+gem "pg"
```

Comment:
can you add your commit `Gemfile.lock`? I have a few changes locally but I'm not sure which ones are needed here